src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -98,44 +99,38 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
     String url = videoUrl.get();
 
-    Query query = new Query(Lecture.ENTITY_KIND);
+    Query query = new Query(LectureUtil.KIND);
     PreparedQuery results = datastore.prepare(query);
     Iterable<Entity> resultsIterable = results.asIterable();
 
     for (Entity lecture : resultsIterable) {
-      if (lecture.getProperty(Lecture.PROP_VIDEO_URL).equals(url)) {
+      if (lecture.getProperty(LectureUtil.VIDEO_URL).equals(url)) {
         return Optional.of(lecture);
       }
     }
     return Optional.empty();
   }
 
   @VisibleForTesting
-  /** Creates and returns {@code lectureEntity} using parameters found in {@code request}. */
+  /** Returns {@code lectureEntity} using parameters found in {@code request}. */
   // TODO: Send errors as a response if fields are empty.
-  // TODO: Move to LectureEntityCreator class.
-  protected Entity createLectureEntity(HttpServletRequest request) {
+  protected Entity getLectureEntity(HttpServletRequest request) {
```

Comment:
Can this be renamed to `getLectureFromRequest`?

Reply:
or `getLectureEntityFromRequest`

Reply:
yes